assembly/compliance-tool/src/test/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/SW360TestUtils.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2020.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.frontend.compliancetool.sw360;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.eclipse.sw360.antenna.model.coordinates.Coordinate;
+import org.eclipse.sw360.antenna.sw360.rest.resource.LinkObjects;
+import org.eclipse.sw360.antenna.sw360.rest.resource.Self;
+import org.eclipse.sw360.antenna.sw360.rest.resource.components.SW360Component;
+import org.eclipse.sw360.antenna.sw360.rest.resource.components.SW360ComponentEmbedded;
+import org.eclipse.sw360.antenna.sw360.rest.resource.components.SW360ComponentType;
+import org.eclipse.sw360.antenna.sw360.rest.resource.components.SW360SparseComponent;
+import org.eclipse.sw360.antenna.sw360.rest.resource.releases.SW360Release;
+import org.eclipse.sw360.antenna.sw360.rest.resource.releases.SW360SparseRelease;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+
+public class SW360TestUtils {
```

Comment:
This utils class creates hard coded objects, so no other test really can use this. Try to parameterise.